modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -64,9 +64,18 @@ The *Add pipeline* checkbox is checked and *Configure PAC* is selected by defaul
 * Pipeline operator is installed
 * `pipelines-as-code` is enabled
 * `.tekton` directory is detected in the Git repository
-
 ====
 
+
+. Add a webhook to your repository. If *Configure PAC* is checked and the GitHub App is set up, you can see the *Use GitHub App* and *Setup a webhook* options. If GitHub App is not set up, you can only see the *Setup a webhook* option:
+
+.. Go to *Settings* -> *Webhooks* and click *Add webhook*.
+.. Set the *Payload URL* to the Pipelines as Code controller public URL.
+.. Select the content type as *application/json*.
+.. Add a webhook secret and note it in an alternate location. With `openssl` installed on your local machine, generate a random secret.
```

Comment:
Neutral question: Would it be clearer to have these as two separate steps or together as they are?

Reply:
These steps have been taken from another section mentioned in the Pipelines section. I'm guessing we can keep this in the same step as the purpose of this step is to add/generate a secret.